src/styles/index.scss

```diff
@@ -1,3 +1,215 @@
-body {
+$cardFont: Roboto, Arial, sans-serif;
+$defaultFontWeight: 400;
+$accentColor: #060b35;
+$secondaryColor: #616070;
+$moyoBlue: #00acdc;
+$transitionTime: 300ms;
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You've defined a font-family in your html selector here, but you've also defined a $cardFont variable at the top of your file. If these are intended to be the same, you should replace the font-family here with your $cardFont variable for consistency.